### PR TITLE
Add managing-state

### PR DESCRIPTION
**Pull request by Songycs**
[managing-state](https://beta.reactjs.org/learn/managing-state)를 추가합니다.

### CODE REVIEW

File: docs/Managing State/managing-state.mdx [hunk 1]
```diff
@@ -0,0 +1,155 @@
+import { Code } from '../../../src/components/Code';
+import { BlockQuote } from '../../../src/components/BlockQuote/BlockQuote';
+
+# 상태 관리하기
```
Comment (ChipmunkForLove) [inline docs/Managing State/managing-state.mdx]:
```suggestion
# 상태 관리하기 (중급)
```

File: docs/Managing State/managing-state.mdx [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+여러분의 앱이 성장할수록, 여러분의 상태를 조직하고 컴포넌트간의 데이터 흐름을 더 의도적으로 구성하는 것은
+도움이 될 것입니다. 불필요하거나 중복되는 버그의 흔한 이유입니다. 이 챕터에서, 여러분은 상태를 어떻게 잘 구조화하고, 상태 업데이트 로직을
+유지보수 가능하게 하며, 거리가 먼 컴포넌트 사이에 상태를 공유하는 법을 배울 것 입니다.
+
+:::note In this chapter
```
Comment (ChipmunkForLove) [inline docs/Managing State/managing-state.mdx]: 한국어로 바꿔쓰기

File: docs/Managing State/managing-state.mdx [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+:::
+
+## 상태로 입력에 반응하기
+
+React와 함께라면, 여러분은 UI를 코드에서 직접 수정하지 않을 것 입니다. 예를들어 "버튼 비활성화", "버튼 활성화", "성공 메시지 출력" 등의 명령을 작성하지 않을 것 입니다.
+대신에 여러분은 UI에게 컴포넌트의 다른 시각적 상태에 대해 설명할 것이고 ("초기 상태", "타이핑 상태", "성공 상태), 유저 입력에 반응하여 상태 변화를 트리거할 것 입니다.
+이 것은 디자이너가 UI에 대해 생각하는 방식과 유사합니다.
+
+여기 React로 구성된 퀴즈 폼이 있습니다. `status` 상태 값을 제출버튼을 활성화, 혹은 비활성화 하는 것과 성공 메시지를 보여줄지를 결정하는데 어떻게 활용하는지 기록해보세요.
```
Comment (ChipmunkForLove) [inline docs/Managing State/managing-state.mdx:28]:
```suggestion
여기 React로 구성된 퀴즈 폼이 있습니다. `status` 상태 변수를 제출버튼을 활성화, 혹은 비활성화 하는 것과 성공 메시지를 보여줄지를 결정하는데 어떻게 활용하는지 기록해보세요.
```

File: docs/Managing State/managing-state.mdx [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+  link="https://beta.reactjs.org/learn/reacting-to-input-with-state"
+/>
+
+## 상태 구조 선택하기
+
+상태를 잘 구조화하는 것은 수정과 디버깅하기 쉽게하고, 거듭되는 버그의 원인이 되는 컴포넌트의 다른 점을 만들어냅니다.
+가장 중요한 원칙은 상태가 불필요하거나 중복되는 정보를 가지게해서는 안된다는 것 입니다. 불필요한 상태가 있다면, 업데이트하는 것을 잊기 쉬워
+버그를 만들 것 입니다.
+
+예를 들어, 이 폼은 불필요한 `fullName` 상태값을 가집니다.
```
Comment (ChipmunkForLove) [inline docs/Managing State/managing-state.mdx]:
```suggestion
예를 들어, 이 폼은 *중복되는* `fullName` 상태값을 가집니다.
```

File: docs/Managing State/managing-state.mdx [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+<Card
+  title="Ready to learn this topic?"
+  description="리듀서로 상태 로직을 추출하기를 읽고 리듀서 함수에 로직을 통합하는 법을 배워보세요"
+  link="https://beta.reactjs.org/learn/extracting-state-logic-into-a-reducer"
+/>
+
+## 컨텍스트로 데이터를 깊이 전달하기
+
+흔치 않지만, 여러분은 부모 컴포넌트에서 자식 컴포넌트로 props를 통해 정보를 넘겨주게 됩니다.
```
Comment (ChipmunkForLove) [inline docs/Managing State/managing-state.mdx]:
```suggestion
일반적으로, 여러분은 부모 컴포넌트에서 자식 컴포넌트로 props를 통해 정보를 넘겨주게 됩니다.
```

**Comment by Songycs** [comment]
[8c06223](https://github.com/LINE-NEXT-LEVEL/client/pull/17/commits/8c062230e0fb9cdb7c3beae490637ad2c2b8bf2f) 
피드백 반영되었습니다

**Review by ChipmunkForLove — APPROVED** [review]
👍👍👍

**Pull request merged**